packages/seeder/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
81],[162,181],[164,182],[142,2],[144,183],[143,2],[166,184],[165,2],[167,185],[169,171],[171,186],[168,171],[170,171],[174,171],[172,2],[176,171],[173,171],[178,187],[177,171],[175,171],[219,188],[181,189],[184,190],[185,190],[182,190],[186,191],[180,192],[179,2],[183,189],[188,193],[187,171],[197,2],[199,194],[200,195],[198,2],[202,196],[211,2],[207,171],[204,196],[212,197],[214,197],[213,197],[203,196],[206,196],[205,171],[209,171],[215,198],[201,2],[208,171],[210,196],[217,199],[218,200],[216,2],[137,2],[138,171],[136,171],[141,201],[139,202],[140,203],[190,204],[193,171],[192,171],[194,204],[191,204],[196,205],[195,204],[189,2],[100,206],[98,207],[101,208],[102,209],[103,210],[104,211],[105,212],[106,213],[107,214],[108,215],[109,216],[124,217],[111,218],[123,2],[110,219],[112,220],[113,221],[114,222],[115,223],[116,224],[117,225],[118,226],[119,227],[120,228],[121,229],[122,230],[8,2],[9,2],[13,2],[12,2],[2,2],[14,2],[15,2],[16,2],[17,2],[18,2],[19,2],[20,2],[21,2],[3,2],[4,2],[45,2],[25,2],[22,2],[23,2],[24,2],[26,2],[27,2],[28,2],[5,2],[29,2],[30,2],[31,2],[32,2],[6,2],[33,2],[34,2],[35,2],[36,2],[7,2],[37,2],[42,2],[43,2],[38,2],[39,2],[40,2],[41,2],[1,2],[44,2],[11,2],[10,2],[127,236],[220,237],[222,237]],"semanticDiagnosticsPerFile":[228,226,224,225,231,227,229,230,232,234,235,236,240,233,241,242,243,248,249,251,256,252,255,254,239,247,257,258,363,269,335,334,333,274,290,288,289,275,358,260,262,263,264,267,270,287,265,282,268,283,286,284,281,261,266,285,291,279,273,271,280,277,276,272,278,354,348,341,340,349,350,342,355,336,337,338,357,339,343,344,351,352,353,356,345,292,346,347,332,330,331,296,297,298,299,300,301,302,303,322,304,305,306,307,308,309,329,310,311,312,327,313,328,314,325,326,315,316,317,323,324,318,319,320,321,295,294,293,259,364,359,360,365,366,367,368,378,253,379,380,244,361,362,382,383,381,384,385,386,387,388,389,390,391,392,393,394,245,395,396,250,397,46,47,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,48,95,64,65,66,96,67,68,69,70,71,72,73,74,75,76,77,78,80,79,81,82,83,84,85,86,87,88,89,90,91,92,93,94,411,398,405,401,399,402,406,407,404,403,408,409,410,400,412,413,414,417,416,415,418,419,420,238,237,427,425,426,430,428,429,421,424,431,432,433,423,442,443,246,444,445,446,447,449,448,451,450,441,434,435,436,437,438,440,439,452,453,454,455,456,97,369,422,99,370,374,376,375,373,377,372,371,147,148,145,146,130,129,134,131,133,135,132,152,151,150,149,157,154,156,153,158,155,159,160,161,163,162,164,142,144,143,166,165,167,169,171,168,170,174,172,176,173,178,177,175,219,181,184,185,182,186,180,179,183,188,187,197,199,200,198,202,211,207,204,212,214,213,203,206,205,209,215,201,208,210,217,218,216,137,138,136,141,139,140,190,193,192,194,191,196,195,189,100,98,101,102,103,104,105,106,107,108,109,124,111,123,110,112,113,114,115,116,117,118,119,120,121,122,8,9,13,12,2,14,15,16,17,18,19,20,21,3,4,45,25,22,23,24,26,27,28,5,29,30,31,32,6,33,34,35,36,7,37,42,43,38,39,40,41,1,44,11,10,127,128,220,221,126,222,223,125]},"version":"4.7.2"}
```

Comment:
what's the purpose of this file? if not necessary then it should be removed

Reply:
Removed because of not necessary right now